smarts/core/vehicle.py

```diff
@@ -249,8 +238,9 @@ def sensors(self) -> dict:
         return self._sensors
 
     @property
-    def renderer(self) -> Optional[Renderer]:
+    def renderer(self):  # type: ignore
```

Comment:
Ideally here we would pull out a generalised `Renderer` interface and use that instead.